draft-ietf-secevent-subject-identifiers.md

```diff
@@ -461,6 +480,13 @@ Defining Document(s)
 * Change Controller: IETF
 * Defining Document(s): {{sub-ids}} of this document.
 
+#### Decentralized Identifier Format
+
+* Type Name: `did`
```

Comment:
Should this be `did` or `did_url`? If there is a reasonable chance that we will want to define a subject identifier format that specifically allows only bare DIDs, then I think `did_url` would be better. But my sense is that a profiling spec/implementer could use this format and define constraints on the types of DID URLs that are allowed/accepted. (or those constraints may be implied by the context) If so, then leaving this as `did` makes sense. But I'd appreciate confirmation from someone more familiar with the DID world.

Reply:
It should be `did`. Profiles can constrain if they want already and a new format could be defined for bare DIDs if necessary. All DID URLs are usable as DIDs, and the DID WG decided that the top level component should be the more flexible format.

Reply:
> All DID URLs are usable as DIDs

Interesting. That raises another question: is the subject identified by a `did` subject identifier the DID Subject identified by the bare DID portion of the DID URL in the `url` property, or is it the entity/resource that the DID URL resolves to? For example, suppose Alice has a DID URL that resolves to a specific verification method described within a DID Document for which Alice is the DID Subject. If Alice used this DID URL in a `did` subject identifier, should that be interpreted as identifying Alice, or the verification method? Or does it depend on context? Either way, a non-normative clarification would be helpful, I think. I can write that, just need to know which way to write it.